src/workloads/execution/TextIndex.yml

```diff
@@ -0,0 +1,42 @@
+# This workload inserts ~1GB of documents in the first phase, and creates a text index on them in the second.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 500000
+    BatchSize: 10000
+    Document:
+      a: {^RandomString: {length: 1000}}
+      b: {^RandomString: {length: 1000}}
+  - &Nop {Nop: true}
+
+- Name: IndexCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Operation:
+      OperationMetricsName: CreateIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            a: "text"
+            b: "text"
+          name: textIndex
+          weights:
+            a: {^RandomInt: {distribution: geometric, p: 0.1}}
```

Comment:
Whoa -- I had to look this one up: https://docs.mongodb.com/manual/core/index-text/#specify-weights

Although, if this is doing one index collection call, I think you might be better off putting in actual numbers here. Super cool though if you do the index creation multiple times.